src/Ledger/Certs/Properties.agda

```diff
@@ -131,3 +133,115 @@ instance
 
 Computational-CERTS : Computational _⊢_⇀⦇_,CERTS⦈_ String
 Computational-CERTS = it
+
+private variable
+  dCert : DCert
+  Γ : CertEnv
+  l : List DCert
+
+instance
+  _ = +-0-monoid
+
+  HasCoin-Map : ∀ {A} → ⦃ DecEq A ⦄ → HasCoin (A ⇀ Coin)
+  HasCoin-Map .getCoin s = ∑[ x ← s ] x
+
+≡ᵉ-getCoin : ∀ {A} → ⦃ _ : DecEq A ⦄ → (s s' : A ⇀ Coin) → s ˢ ≡ᵉ s' ˢ → getCoin s ≡ getCoin s'
+≡ᵉ-getCoin {A} ⦃ decEqA ⦄ s s' s≡s' = indexedSumᵛ'-cong {C = Coin} {x = s} {y = s'} s≡s'
+
+module _ {A : Type} ⦃ _ : DecEq A ⦄
+  where
+  getCoin-singleton : ((a , c) : A × Coin) → indexedSumᵛ' id ❴ (a , c) ❵ ≡ c
+  getCoin-singleton _ = indexedSum-singleton' ⦃ M = +-0-commutativeMonoid ⦄ (finiteness _)
+
+  module _ {gc-hom : (d₁ d₂ : A ⇀ Coin) → getCoin (d₁ ∪ˡ d₂) ≡ getCoin d₁ + getCoin d₂} where
+    ∪ˡsingleton≡ : {m : A ⇀ Coin} {(a , c) : A × Coin} → getCoin (m ∪ˡ ❴ (a , c) ❵ᵐ) ≡ getCoin m + c
+    ∪ˡsingleton≡ {m} {(a , c)} = let open ≡-Reasoning in begin
+      getCoin (m ∪ˡ ❴ (a , c) ❵)
+        ≡⟨ gc-hom m ❴ (a , c) ❵ ⟩
+      getCoin m + getCoin{A = A ⇀ Coin} ❴ (a , c) ❵
+        ≡⟨ cong (getCoin m +_) (getCoin-singleton (a , c)) ⟩
+      getCoin m + c ∎
+
+
+module _ {gc-hom : (d₁ d₂ : Credential ⇀ Coin) → getCoin (d₁ ∪ˡ d₂) ≡ getCoin d₁ + getCoin d₂}
```

Comment:
This assumption is wrong, it only holds if the two maps have disjoint sets of keys. In particular the type of `CERTBASE-pov` is wrong because one of the sets of keys will actually always be a subset of the other (in practice).